src/ApiPort.VisualStudio/ApiPortVSPackage.cs

```diff
@@ -68,7 +68,7 @@ protected override void Initialize()
                 mcs.AddCommand(analyzeMenuOptionsItem);
 
                 CommandID analyzeMenuToolbarCommandID = new CommandID(Guids.AnalyzeMenuItemCmdSet, (int)PkgCmdIDList.CmdIdAnalyzeToolbarMenuItem);
-                MenuCommand analyzeMenuToolbarItem = new MenuCommand((_, __) => ShowToolbar(), analyzeMenuToolbarCommandID);
+                MenuCommand analyzeMenuToolbarItem = new MenuCommand(async (_, __) => await ShowToolbarAsync(), analyzeMenuToolbarCommandID);
```

Comment:
Does the `ShowToolbarAsync` need to be `async Task`? If it is only used as an event handler, you can just use `async void`